keg_auth/forms.py

```diff
@@ -148,7 +152,16 @@ def html_link(obj):
             flask.url_for(endpoint, objid=obj.id)
         )
 
+    def filter_disabled_utc(date):
+        if isinstance(date, dt.date):
+            date = arrow.get(date)
+
+        return date
+
     class User(PermissionsMixin, BundlesMixin, GroupsMixin, ModelForm):
+        disabled_utc = DateField('Disable Date', [validators.Optional()],
```

Comment:
I notice that this isn't in the fields param above, but it is in the override in the test app. Does the default here work? I'd think that there may be an error since it's not included in the field order.

As to what the field order should be, we should have the Enabled checkbox first, then the date field.

Reply:
I guess the idea here is that the developer would have to opt in to the disable date input on the user form so it's not there by default. Any fields not included in the `field_order` are just omitted from the form so there's no error.

The developer would have to specify `fields=['is_enabled', 'disabled_utc']`. I suppose it might be better to have it there by default and the dev can opt out. Most apps end up overriding the `fields` argument anyway to provide additional fields like `name` in which case they are basically forced to opt in/out of the default fields.

Reply:
Is something overriding the behavior at https://github.com/level12/keg-elements/blob/master/keg_elements/forms/__init__.py#L424?

Reply:
Yes. The User form has overridden `__iter__` which is where the `field_order` stuff takes place. Actually the user form uses `field_order` where the keg-elements form expects `_field_order` so it wouldn't do anything either way.

I guess the main difference in these implementations of `__iter__` is the user form omits fields not specified while the keg-elements form expects all fields to be specified and throws an error otherwise. I think it works this way so keg-auth can provide some default fields for the user form but the developer can opt to hide them.

Maybe it would be better to use a different argument like `exclude` for hiding fields and use the `fields` argument only for adding new fields.

Reply:
That's funny... I mean, I'm probably the one that wrote it that way and probably for that reason. But I agree with the latest change - the field is there, go ahead and expose it, and developers can hide it if there's some reason to.